tests/integration/pilot/multicluster_test.go

```diff
@@ -91,6 +95,82 @@ serviceSettings:
 		})
 }
 
+func TestBadRemoteSecret(t *testing.T) {
+	framework.NewTest(t).
+		RequiresMinClusters(2).
+		Features(
+			// TODO tracking topologies as feature labels doesn't make sense
+			"installation.multicluster.multimaster",
+			"installation.multicluster.remote",
+		).
+		Run(func(t framework.TestContext) {
+			// we don't need to test this per-cluster
+			primary := t.Clusters().Configs()[0]
+			// it doesn't matter if the other cluster is a primary/remote/etc.
+			remote := t.Clusters().Exclude(primary)[0]
+
+			var (
+				ns  = i.Settings().SystemNamespace
+				sa  = "istio-reader-no-perms"
+				pod = "istiod-bad-secrets-test"
+			)
+			t.Logf("creating service account %s/%s", ns, sa)
+			if _, err := remote.CoreV1().ServiceAccounts(ns).Create(context.TODO(), &corev1.ServiceAccount{
+				ObjectMeta: metav1.ObjectMeta{Name: sa},
+			}, metav1.CreateOptions{}); err != nil {
+				t.Fatal(err)
+			}
+
+			// intentionally not doing this with subtests since it would be pretty slow
+			for name, opts := range map[string][]string{
+				"unreachable server": {"--name", "unreachable", "--server", "https://255.255.255.255"},
+				"no permissions":     {"--name", "no-permissions", "--service-account", "istio-reader-no-perms"},
+			} {
+				secret, err := istio.CreateRemoteSecret(t, remote, i.Settings(), opts...)
+				if err != nil {
+					t.Fatalf("failed generating secret with %s: %v", name, err)
+				}
+				t.Config().ApplyYAMLOrFail(t, ns, secret)
+			}
+
+			// create a new istiod pod using the template from the deployment, but not managed by the deployment
```

Comment:
Any reason to do this vs scale the deployment?

Reply:
We can control it a bit more directly, watch/delete the exact instance we care about and there should be less lag in k8s for the new Pod to come up.